tournaments/index.js

```diff
@@ -996,13 +996,13 @@ class Tournament {
 
 		delete exports.tournaments[this.room.id];
 		delete this.room.game;
-		if (this.room.isOfficial) {
-			for (let i in this.players) {
-				Users(this.players[i].userid).tourBoost = false;
-				Users(this.players[i].userid).gameBoost = false;
+		for (let i in this.players) {
+			Users(this.players[i].userid).tourBoost = false;
```

Comment:
Toss these in the official thing as well

Reply:
ah true